src/utils/components/FloatingPosition.jsx

```diff
@@ -128,6 +129,8 @@ class FloatingPosition extends React.PureComponent {
 	}
 
 	render() {
+		const throttledMaxWidth = this.props.media.isAtSmallUp ? '350px' : '200px';
```

Comment:
"throttle" implies handling something like a stream of events... `boundedMaxWidth` may be more descriptive of what this render prop provides.

Reply:
just `maxWidth` would also work here; it's probably not necessary to expose _how_ at works as much as what the prop value should be used for.

Reply:
I'm going to use `boundedMaxWidth` so I can still do `maxWidth={maxWidth || boundedMaxWidth}` in `Dropdown` and `Toggle` and let people override what gets passed through `FloatingPosition`